user-service/src/main/resources/application.yaml

```diff
@@ -50,17 +42,15 @@ management:
     shutdown:
       enabled: true
 
-feign:
-  client:
-    config:
-      default:
-        connectTimeout: 5000
-        readTimeout: 5000
-        loggerLevel: basic
-    default-config: default
-
-user-service:
-  implementation: null-user-service
```

Comment:
Do we no longer need this selector?

Reply:
I think we still need this so have added it back in.